llvm/lib/Target/AMDGPU/AMDGPURegisterBankInfo.cpp

```diff
@@ -3101,6 +3101,24 @@ void AMDGPURegisterBankInfo::applyMappingImpl(
       applyDefaultMapping(OpdMapper);
       constrainOpWithReadfirstlane(B, MI, 8); // M0
       return;
+    case Intrinsic::prefetch: {
```

Comment:
This should have a G_ instruction.

Why erase here and not in the legalizer?

Reply:
It should have G_PREFETCH, but we do not currently have it.
Then I honestly do not remember why do I erase it here. Likely because I am erasing it here for VGPRBank anyway.